tests/pages/services/TasksTable-cy.js

```diff
@@ -87,4 +87,47 @@ describe("Tasks Table", function() {
       });
     });
   });
+
+  context("Service tasks checkbox", function() {
```

Comment:
Let's also add a test to check the `stop` and `restart` buttons appear above the table when a task is checked.